plugins/src/main/kotlin/com/freeletics/gradle/plugin/FreeleticsBaseExtension.kt

```diff
@@ -32,6 +32,10 @@ public abstract class FreeleticsBaseExtension(private val project: Project) : Ex
         project.configureMoshi(sealed)
     }
 
+    public fun useSerialization() {
+        project.plugins.apply("org.jetbrains.kotlin.plugin.serialization")
```

Comment:
Probably it would make sense to also add `org.jetbrains.kotlinx:kotlinx-serialization-core` to dependencies, maybe even `kotlinx-serialization-json`. What do you think?

Reply:
Added core: [5fe9854](https://github.com/freeletics/freeletics-gradle-plugins/pull/301/commits/5fe9854a961d42f8f0d0aa21b6ac5b8a5080dcb8) I decided against json since you won't need that dependency in modules where you define classes, only where you actively parse/serialize json.